msal/src/Microsoft.Identity.Client/Exceptions/MsalClientException.cs

```diff
@@ -30,72 +30,120 @@
 namespace Microsoft.Identity.Client
 {
     /// <summary>
-    /// This exception class represents errors that are local to the library or the device.
+    /// This exception class represents errors that are local to the library or the device. contrary to
+    /// <see cref="MsalServiceException"/> which represent errors happening from the Azure AD service or
+    /// the network
```

Comment:
perhaps this should be a see also? + an aka.ms link to wiki page on exceptions

Reply:
Brilliant idea. Done. Thanks!